### PR TITLE
Rephrase dependency injection-inversion questions to make them clearer

### PR DESCRIPTION
**Pull request by mikeger**
I know that DI is important (right, @ilyapuchka ?), but this seems to be a mere copy-paste issue.

### CODE REVIEW

File: Interview/questions.md [hunk 1]
```diff
@@ -28,7 +28,6 @@ Please keep in mind that this document is in a continuous state of flux, with ne
 - Are you familiar with `Design Patterns` and the `Gang of Four` book?
 - What is your opinion on the use of the `Singleton` pattern? What are its pros and cons?
 - Are you familiar with `Dependency Inversion`? If so, what is it good for?
```
Comment (diegopetrucci) [inline Interview/questions.md]:
```suggestion
- Are you familiar with `Dependency Inversion`? If so, what is it good for?
- And what about the `inversion` principle?
```

**Comment by mikeger** [comment]
EDIT: So yeah, I cannot read properly, it's definitely not a copy-paste. Feel free to adjust it in the best possible way in order to avoid the confusion for the people like me.

**Comment by diegopetrucci** [comment]
@mikeger your point was valid, if the difference is not clear for you it means it's not clear for other either. I would merge this with the updated wording, it's a good change.

**Review by diegopetrucci — APPROVED** [review]

**Comment by ilyapuchka** [comment]
Hi @mikeger ! =)

File: Interview/questions.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ Please keep in mind that this document is in a continuous state of flux, with ne
...
-- Are you familiar with `Dependency Injection`? If so, what is it good for?
+- And what about the `inversion` principle?
```
Comment (ilyapuchka) [inline Interview/questions.md]:
```suggestion
- And what about the `Dependency Inversion` principle?
```

**Comment by mluisbrown** [comment]
`Dependency Injection` and `Dependency Injection` are 2 distinct things. IMO the current state of the document is correct. So I would move to close this PR again @diegopetrucci.

**Comment by mikeger** [comment]
@mluisbrown 
Screenshot 2019-07-02 at 13 50 14
this is exactly what I mean: easy to confuse, so that it can even happen for the people who know the matter.

File: Interview/questions.md (hunk 2 above)
Comment (mikeger) [reply Interview/questions.md]: With this variant both lines 30 and 31 are talking about the in**v**ersion, let me correct the line 30.

Comment (ilyapuchka) [reply Interview/questions.md]: ah, right, missed the previous question

**Review by ilyapuchka — APPROVED** [review]

**Comment by mluisbrown** [comment]
@mikeger it was a copy and paste error, not confusion.

The original document has: 
```
Are you familiar with `Dependency Inversion`? If so, what is it good for?		
Are you familiar with `Dependency Injection`? If so, what is it good for?
```

So IMO the change is not really necessary, but I don't oppose it either, so it's fine.

**Review by mluisbrown — APPROVED** [review]

**Pull request merged**